ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/deprovision/cucushift-installer-rehearse-aws-ipi-deprovision-chain.yaml

```diff
@@ -5,6 +5,7 @@ chain:
   - chain: gather-core-dump
   - ref: gather-aws-console
   - chain: ipi-deprovision
+  - ref: ipi-conf-aws-oidc-creds-deprovision
```

Comment:
Personally I prefer to create dedicated deprovision chain when oidc enabled instead of injecting the step to a common ipi deprovision chain, to keep this deprovision chain tidy enough.

Reply:
the `ipi-conf-aws-oidc-creds-deprovision` must be performed between _destroy cluster_ and _remove IAM user (ccoctl)_, that's why put it in `ipi-deprovision`

Reply:
Can we split aws-ipi-deprovision chain to inject `ipi-conf-aws-oidc-creds-deprovision` for those install workflow ?

Reply:
Once that, we also can save the change of https://github.com/openshift/release/pull/61415/files#diff-245ade92dcc585c110a230d5b6cdbdab21d8b4684f8f8279187fdaedb1d3d6fcR8

Reply:
> Can we split aws-ipi-deprovision chain to inject `ipi-conf-aws-oidc-creds-deprovision` for those install workflow ?

Keep `cucushift-installer-rehearse-aws-ipi-deprovision` unchanged, and create a new `cucushift-installer-rehearse-aws-ipi-sts-deprovision` chain?:
```
  steps:
  - chain: gather-network
  - chain: gather-core-dump
  - ref: gather-aws-console
  - chain: ipi-deprovision
  - ref: ipi-conf-aws-oidc-creds-deprovision
  - ref: aws-deprovision-users-and-policies
```

Reply:
Yes.